config/locales/exceptions.en.yml

```diff
@@ -486,6 +486,12 @@ en:
         code: 'VAOS_400'
         detail: # deliberarly left nil to reference response.body which is unparsable string (not json)
         status: 400
+      VAOS_204_hack:
```

Comment:
"hack"? No real context for why that's named like that. There's no other VAOS_204 so maybe some more explanation

Reply:
I think we can probably just call it VAOS_204. 

What's happening here isn't even really a hack, I can fix this. Basically the host is returning a 204 which would otherwise be interpreted as a success. What they probably should be returning is a 400 or 404 because the parent association of the message is invalid or not found.

Reply:
Yeah the pattern here can be followed - `VAOS_204` is fine imo. There are other examples of this